lib/emscripten/src/syscalls/unix/host_fs.rs

```diff
@@ -668,129 +496,84 @@ pub fn ___syscall181(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> c_in
         assert_eq!(zero, 0);
     }
     let offset: i64 = varargs.get(ctx);
-
     let buf_ptr = emscripten_memory_pointer!(ctx.memory(0), buf) as _;
-    let status = unsafe { pwrite(fd, buf_ptr, count as _, offset) as _ };
+    let status = unsafe { libc::pwrite(fd, buf_ptr, count as _, offset) as _ };
     debug!(
         "=> fd: {}, buf: {}, count: {}, offset: {} = status:{}",
         fd, buf, count, offset, status
     );
     status
 }
 
-/// fchmod
-pub fn ___syscall94(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> c_int {
-    debug!("emscripten::___syscall118 (fchmod) {}", _which);
-    let fd: c_int = varargs.get(ctx);
-    let mode: mode_t = varargs.get(ctx);
-    unsafe { fchmod(fd, mode) }
-}
+/// stat64
+pub fn ___syscall195(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> c_int {
+    debug!("emscripten::___syscall195 (stat64) {}", _which);
+    let pathname: u32 = varargs.get(ctx);
+    let buf: u32 = varargs.get(ctx);
 
-/// wait4
-#[allow(clippy::cast_ptr_alignment)]
-pub fn ___syscall114(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> pid_t {
-    debug!("emscripten::___syscall114 (wait4)");
-    let pid: pid_t = varargs.get(ctx);
-    let status: u32 = varargs.get(ctx);
-    let options: c_int = varargs.get(ctx);
-    let rusage: u32 = varargs.get(ctx);
-    let status_addr = emscripten_memory_pointer!(ctx.memory(0), status) as *mut c_int;
-
-    let rusage_addr = emscripten_memory_pointer!(ctx.memory(0), rusage) as *mut rusage;
-    let res = unsafe { wait4(pid, status_addr, options, rusage_addr) };
-    debug!(
-        "=> pid: {}, status: {:?}, options: {}, rusage: {:?} = pid: {}",
-        pid, status_addr, options, rusage_addr, res
-    );
-    res
-}
+    let pathname_addr = emscripten_memory_pointer!(ctx.memory(0), pathname) as *const i8;
 
-/// fsync
-pub fn ___syscall118(ctx: &mut Ctx, _which: c_int, mut varargs: VarArgs) -> c_int {
-    debug!("emscripten::___syscall118 (fsync) {}", _which);
-    let fd: c_int = varargs.get(ctx);
-    unsafe { fsync(fd) }
+    unsafe {
+        let mut _stat: libc::stat = std::mem::zeroed();
+        let ret = libc::stat(pathname_addr, &mut _stat);
+        debug!("ret: {}", ret);
```

Comment:
I added some extra info in this debug statement, might be good to keep around (I'm not sure if the changes weren't migrated yet, were lost in migration, or were intentionally removed)
```
debug!(
            "=> pathname: {}, buf: {}, path: {} = {}\nlast os error: {}",
            pathname,
            buf,
            std::ffi::CStr::from_ptr(pathname_addr).to_str().unwrap(),
            ret,
            Error::last_os_error()
        );
```
is what that looks like to me locally